SonataUserBundle.php

```diff
@@ -74,4 +78,29 @@ public function registerFormMapping()
             'sonata_user_api_form_user' => 'Sonata\UserBundle\Form\Type\ApiUserType',
         ));
     }
+
+    /**
+     * @param ContainerBuilder $container
+     */
+    private function addFOS1RegisterMappingsPass(ContainerBuilder $container)
+    {
+        $fosMappingPath = realpath($container->getParameter('kernel.root_dir').'/../vendor/friendsofsymfony/user-bundle/FOS/UserBundle/Resources/config/doctrine');
+
+        if (!is_dir($fosMappingPath)) {
+            return;
+        }
+
+        $mappings = array(
+            $fosMappingPath => 'FOS\UserBundle\Model',
+        );
+        if (class_exists('Doctrine\Bundle\DoctrineBundle\DependencyInjection\Compiler\DoctrineOrmMappingsPass')) {
+            $container->addCompilerPass(DoctrineOrmMappingsPass::createXmlMappingDriver($mappings, array('fos_user.model_manager_name'), 'fos_user.backend_type_orm'));
```

Comment:
i think with FOSUserBundle v1 the parameter `fos_user.backend_type_orm` is not set, so this won't trigger. pick a name that you set in this bundle.